### DISCUSSION TITLE
Lugar indicado para preguntar

### DISCUSSION THREAD

**Discussion by BrunoPCH** [Launch-X-Latam/MisionFrontEnd#455] [Team Posts]
Hola chicos, según lo que yo entendi en el primer video, las preguntas se hacen en el siguiente foro: [https://github.com/Launch-X-Latam/MisionFrontEnd/discussions](https://github.com/Launch-X-Latam/MisionFrontEnd/discussions).
En este foro, al menos a mi me genera muchos correos de notificaciones y creo que el publicar aqui promueve el desorden. ¿Podrían por favor, publicar sus preguntas e inquietudes aca:
https://github.com/Launch-X-Latam/MisionFrontEnd/discussions. ?

Yo tuve problemas para tener acceso a Micorsoft teams, escribí a contacto@innovaccion.mx desde el correo que me dieron por parte de LaunchX y ahí fueron muy amables al atender mis dudas y mi problema. También puede contactarlos por medio de mensaje en https://www.instagram.com/innovaccionvirtual/.

Saludos y buena vibra.

**Comment by Yhor93** [comment]
Me comunique por instagram y por correo es la hora y no tengo respuesta
concreta ya van dos semanas desde que inicio el proyecto Launch y no he
podido asistir a la primera clase ni acceder a las grabaciones T_T
